contrib/dynamic_embedding/src/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_subdirectory(3rd)
```

Comment:
Lets specify which C++ version is it, C++ 17,
set(CMAKE_CXX_STANDARD 17 CACHE STRING "The C++ standard whose features are requested to build this target.")